src/pages/components/menu-buttons/usage.mdx

```diff
@@ -0,0 +1,621 @@
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not narrow the menu to fit with other elements">
+
+![Do not narrow the menu to fit with other elements](images/usage-menu-width-dont-1.png)
+
+  </DoDont>
+</DoDontRow>
+
+<DoDontRow>
+  <DoDont type="do" caption="Do match the width when the menu or combo button label becomes longer">
+
+![Do match the width when the menu or combo button labels become longer](images/usage-menu-width-do-2.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not make the menu appear shorter than the menu or combo buttons">
+
+![Do not make the menu appear shorter than the menu or combo buttons](images/usage-menu-width-dont-2.png)
+
+  </DoDont>
+</DoDontRow>
+
+Ghost buttons only adhere to the
+[button](https://carbondesignsystem.com/components/button/style/) style. This is
+because enlarging the ghost button's width to match the menu's width would
+result in the chevron appearing disconnected from its associated button,
+especially when the menu button is in its closed state.
+
+<DoDontRow>
+  <DoDont type="do" caption="Do use default ghost button in all cases">
+
+![Do use default ghost button in all cases](images/usage-menu-ghost-button-width-do.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not use fluid ghost button in any cases">
+
+![Do not use fluid ghost button in any cases](images/usage-menu-ghost-button-width-dont.png)
+
+  </DoDont>
+</DoDontRow>
+
+### Alignment
+
+#### Default
+
+By default, combo button, menu button, or overflow button remains positioned at
+the top, to the left or right side of the menu when it is open, depending on the
+available space and layout.
+
+<Row>
+<Column colLg={8}>
+
+![Menu default alignments](images/usage-menu-alignment-menu-buttons-default.png)
+
+</Column>
+</Row>
+
+#### Alternatives
+
+Alternatively, open menus can be positioned at the bottom, to the left or right
+of its associated menu button trigger, depending on the available space and
+layout.
+
+<Row>
+<Column colLg={8}>
+
+![Menu alternative alignments](images/usage-menu-alignment-menu-buttons-alternatives.png)
+
+</Column>
+</Row>
+
+### Placement
+
+#### Menu and combo buttons
+
+These buttons are ideal for page headers, where multiple, distinct, actions may
+impact the page content, and where real-estate is limited. In the page header
+component, multiple buttons will become a single action button at smaller screen
+sizes. These buttons should only be used when the actions affect a full page or
+a large object on a page (examples include a canvas UI, diagram or whole data
+table).
+
+<Row>
+<Column colLg={8}>
+
+![Menu button alignment example](images/usage-menu-placement-1.png)
+
+</Column>
+</Row>
+
+#### Overflow menu
+
+When actions affect a smaller proportion of a page, such as a table rows or
+cards, use an overflow menu.
+
+<Row>
+<Column colLg={8}>
+
+![Overflow menu alignment example](images/usage-menu-placement-2.png)
```

Comment:
The overflow menu in this image has a defect with the shadow coming on top where the button and menu meet. The shadow should be behind the button and menu.